test/proxy/beacon/BeaconProxy.test.js

```diff
@@ -34,7 +34,7 @@ describe('BeaconProxy', function () {
     it('non-compliant beacon', async function () {
       const badBeacon = await ethers.deployContract('BadBeaconNoImpl');
 
-      await expect(this.newBeaconProxy(badBeacon, '0x')).to.be.revertedWithoutReason;
+      await expect(this.newBeaconProxy(badBeacon, '0x')).to.be.reverted;
```

Comment:
I guess the issue here is a `ERC1967InvalidImplementation` with address(1). We should test that.

Reply:
```suggestion
      await expect(this.newBeaconProxy(badBeacon, '0x'))
        .to.be.revertedWithCustomError(this.factory, 'ERC1967InvalidImplementation');
```